src/Prima/Pyxis/Form/Autocomplete.elm

```diff
@@ -575,17 +699,18 @@ render model ((State stateConfig) as stateModel) autocompleteModel =
         , pristineAttribute stateModel autocompleteModel
         ]
         [ Html.input
-            (buildAttributes model stateModel autocompleteModel)
+            (List.append (buildAttributes model stateModel autocompleteModel) [ Attrs.autocomplete False ])
```

Comment:
Questo non va usato qui, deve essere configurabile da fuori. Dovrebbe già essere fattibile utilizzando 
`|> withAttribute [autocomplete False]`
In alcuni casi potremmo volere mantenere l'autocomplete del browser

Reply:
Si, effettivamente era una prova che ho dimenticato di rifattorizzare.